### ISSUE TITLE
Migrate site to Docusaurus?

### ISSUE THREAD

**Issue by Daniel15** [yarnpkg/website#858]
Might be worth considering.

cc @yangshun - do you have any experience with converting a Jekyll site to Docusaurus? The Yarn site has some customization around how it does translations though, which might make it a bit tricky.

**Comment by Haroenv** [comment]
since you can use React in Docusaurus, it should be fairly little work for all the search & detail pages that have been done. However, we probably will need to modify the docusaurus design to be closer to the current design to make sure it doesn't stand out that much

**Comment by yangshun** [comment]
@Daniel15 @Haroenv There are a few problems with the current Docusaurus which makes it hard to work with a site like Yarn's because it uses many custom scripts and designs. 

We're currently working on [Docusaurus 2.0](https://github.com/facebook/Docusaurus/issues/789) which will be a fresh rewrite that will make it much easier to work with. I'd advise against moving to Docusaurus for now. 

Stay tuned!

cc @endiliey @JoelMarcey

**Comment by endiliey** [comment]
When the time comes, I will be very happy to offer a helping hand to migrate yarn website to Docusaurus 2.0. 😊

**Comment by JoelMarcey** [comment]
I think https://yarnpkg.com would be a perfect use case to test v2 of Docusaurus.

**Comment by Daniel15** [comment]
Thanks for the info! Sounds great. Let's wait for Docusaurus 2.0 before doing this. Perhaps you could convert Yarn's site as part of the Docusaurus 2.0 launch (ie. it could be an example of a live 2.0 site)?

**Comment by yangshun** [comment]
Yes sounds good. We'll keep Yarn in mind when testing Docusaurus 2.0. Btw who is the owner of Yarn's website now @Daniel15?

**Comment by Daniel15** [comment]
I don't think we have one owner designated for the Yarn site, but it's mostly @Haroenv working on it these days. There's not really much happening with Yarn's site now, other than community pull requests to improve the docs, and the occasional new blog post by the core Yarn team.
